aten/src/ATen/native/native_functions.yaml

```diff
@@ -3772,6 +3772,10 @@
   dispatch:
     CompositeExplicitAutograd: unsqueeze_
 
+# expand_dims, alias of unsqueeze
+- func: expand_dims(Tensor(a) self, int dim) -> Tensor(a)
```

Comment:
What's your thinking on aliasing the inplace version of the operation, unsqueeze_, as well?

Reply:
The in-place version is out of Array API standard, so it does not necessarily need to be supported, but I think it's OK to support it. Done in 83a21ff!

Reply:
Looks great; I made one small tweak to the inplace method docs. I think this is now ready to merge!